docs/getting_started_1.md

```diff
@@ -11,99 +11,109 @@ and run them with JUnit5. **This is enough to use Chutney**.
 [Part 2](getting_started_2.md) will cover how to install and configure a Chutney server
 in order to schedule test runs, keep execution history and browse your execution reports.
 
-# Project Setup
-
 !!! important "Requirements"
 
     * `java` 11 or later
     * your preferred build tool (ex. `maven`, `graddle`, etc.)
+    * your preferred test engine (ex. `Junit 5.x`, `TestNG`, etc.)
```

Comment:
Beware, you may have add files you didn't want to add and commit.

Remove changes to this file and `rebase` your branch from the main.